src/coreclr/vm/jitinterface.cpp

```diff
@@ -5945,6 +5991,27 @@ CorInfoHelpFunc CEEInfo::getUnBoxHelper(CORINFO_CLASS_HANDLE clsHnd)
     return CORINFO_HELP_UNBOX;
 }
 
+/***********************************************************************/
+void* CEEInfo::getRuntimeTypePointer(CORINFO_CLASS_HANDLE clsHnd)
+{
+    LIMITED_METHOD_CONTRACT;
```

Comment:
```suggestion
    CONTRACTL{
        THROWS;
        GC_TRIGGERS;
        MODE_PREEMPTIVE;
    } CONTRACTL_END;
```
`LIMITED_METHOD_CONTRACT;` means that the method does not throw exceptions, allocates or switches GC mode. This does all of it.